overlord/snapstate/backend/setup.go

```diff
@@ -74,8 +74,14 @@ func (b Backend) SetupSnap(snapFilePath, instanceName string, sideInfo *snap.Sid
 		}
 	}
 
+	// in uc20 run mode, all snaps must be copied when installing
```

Comment:
this is a bit too extreme, we are fine with hardlinking when doing normal installs after seeding, we are fine also during seeding except is not applicable. it's symlinks that cross devices that are problematic